clients/driver/handler.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const events = require('../utility');  //not in demo
+const client = require('./index');  //wrong import?
+
+const ready = (payload) => {
```

Comment:
You cannot require './index', because it requires this file, creating a circular dependency. Instead, try passing `client` as an argument in your `ready` function.